src/main/java/org/jboss/ejb/protocol/remote/EJBClientChannel.java

```diff
@@ -237,14 +238,15 @@ public void processInvocation(final EJBReceiverInvocationContext receiverContext
         invocationContext.putAttachment(INV_KEY, invocation);
         final EJBLocator<?> locator = invocationContext.getLocator();
         final int peerIdentityId;
-        if (version >= 3) try {
-            peerIdentityId = channel.getConnection().getPeerIdentityId();
-        } catch (AuthenticationException e) {
-            receiverContext.resultReady(new EJBReceiverInvocationContext.ResultProducer.Failed(e));
-            return;
-        } else {
+        //TODO Elytron
```

Comment:
This should be functional in the latest Remoting/Elytron versions.  Did you encounter a problem?